web-console/future-openshift/search/search.md

```diff
@@ -0,0 +1,46 @@
+# Search page
+The Search page will be updated to have a variety of new features.
+
+## View multiple resources
+
+![2](img/combo2.png)
+* Users can view more than one type of resource by clicking the dropdown and selecting resource types from the list.
+* There will be a max number of resources that users can select at a time, which is yet to be determined.
+
+![3](img/combo3.png)
+* The items will be sorted by name by default.
+* The default columns in the list are `Name`, `Namespace`, `Labels`, `Type`, `Owner`, and `Created`.
+* When multiple types of resources are selected, the primary 'Create' button will not be shown.
+
+![4](img/combo4.png)
+* Users can sort by Type to separate the different resources in the list.
+
+## Bulk select resources
+![5](img/combo5.png)
+* Users will be able to multi select items in the list by clicking their associated checkboxes.
+* To select all at once, users can use the bulk selector.
+
+
+## Take bulk actions
+![6](img/combo6.png)
+* When any items in the list are selected, the actions dropdown will replace the primary action in the toolbar. If there is no primary action, the rest of the toolbar will shift to make room for the dropdown.
+* The primary bulk actions will be `Add Labels`, `Add Annotations`, and `Delete Resources`. More actions may be added in future depending on the resource(s).
```

Comment:
I'm interested to see how Delete Resources works as a bulk action.  Some of our resources require a confirmation with entering the name.  Will those resource types be available for deleting in bulk?

Reply:
Good question! I know deleting namespaces requires this but I can't seem to find other ones. I think that if there are resources that require extra confirmation we can pop a similar modal but have users type "delete" rather than the resource names.
Actually, now I'm curious whether bulk deleting should require confirmation regardless. It could help ensure that users aren't bulk deleting objects by accident. Thoughts?